### ISSUE TITLE
Android port

### ISSUE THREAD

**Issue by LinArcX** [crow-translate/crow-translate#85]
Is it possible to have android version of of this application?

**Comment by Shatur95** [comment]
Yes, I thought about it. This is possible because Qt Widgets support Android applications. But need to adapt interface for touch control. Also need to get rid of selection and popup functionality.
